src/Hydrator/MoneyHydrator.php

```diff
@@ -20,7 +20,7 @@ class MoneyHydrator implements HydratorInterface
     public function extract($object)
     {
         return [
-            'amount' => $object->getAmount(),
+            'amount' => $object->getAmount() / 100,
```

Comment:
First:
I don't know about all currencies, but INR (indian rupee) is definitly not more than 100 units.
The modern rupee is subdivided into 100 paise (singular paisa), see http://en.wikipedia.org/wiki/Indian_rupee.

Second:
Considering dividing 100 or not (with the current module, as it is):
a) You render a form and enter 100 as value and you select USD as currency.
b) money get hydrated and a value of 100 is stored into its money amount property.
c) again 100 is saved to database.
d) last: let's print the value using the given moneyFormat view helper: 1,00 USD ! WTF? I entered 100 USD in a form and get only cents rendered.

Last point: If there is really some currencies out there with more that 100 fractions, it's better to add a getPrecision() method to the currency object itself instead of juggling with divining of 100 in various places.

Best would be, if the module takes care of everything and I don't need to deal with such silly details.